README.md

```diff
@@ -377,9 +272,13 @@ client.query(predicateExpr) { [weak self] result in
 
 ### Example project
 
-`FaunaDB.workspace` contains an Example app that shows the basics of how to use the driver and populates an iOS UITableView with fauna db data as well as paginate and filter fauna db data. It shows how to create a database, create classes, indexes, performs queries and handle queries results.
+`FaunaDB.workspace` contains an iOS Example app that shows the basics of how to use the driver and populates an UITableView with Fauna DB data as well as paginate and filter it. It shows how to create a database, create classes, indexes, performs queries and handle queries results.
+
+You can run the `Example` iOS app in the iPhone simulator by following these steps...
 
-You can run the `Example` iOS app in the iPhone simulator by just cloning this repository, opening the `FaunaDB.workspace` using Xcode and then running `Example` app.
+1. Clone this repository.
+2. Run `carthage update` from the repository root folder.
```

Comment:
Is Carthage preferred for some reason? Could we add the analog command for CocoaPods?